frontend/src/Metamaps/Views/TopicCard.js

```diff
@@ -0,0 +1,61 @@
+/* global $ */
+
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import Active from '../Active'
+import DataModel from '../DataModel'
+import GlobalUI from '../GlobalUI'
+import Mapper from '../Mapper'
+import Router from '../Router'
+import Visualize from '../Visualize'
+
+import ReactTopicCard from '../../components/TopicCard'
+
+const TopicCard = {
+  openTopicCard: null, // stores the topic that's currently open
+  init: function() {
+    // initialize topic card draggability and resizability
+    $('.showcard').draggable({
+      handle: '.metacodeImage',
+      stop: function() {
+        $(this).height('auto')
+      }
+    })
+  },
```

Comment:
yeah, but it probably doesn't need to have both divs... I'll scrap the inner one. I intend to implement dragging, but it might work a bit different, and not use this code. I am doing a little bit of simple design exploration alongside this work